src/routes/pages.routes.js

```diff
@@ -1,139 +1,111 @@
-import { Router } from 'express';
-import { loadMovies, loadMovie } from '../helpers/movies.js';
-import { marked } from 'marked';
-import menu from '../../menu.js';
+import { Router } from "express";
```

Comment:
There are a lot of changes in this file and others, that seems to be mostly unrelated to the work that you've done. Avoid contaminating your PR with unrelated changes by making sure everyone in the team uses auto formatting withthe same editor settings.